transports/janus_http.c

```diff
@@ -1297,6 +1297,49 @@ static int janus_http_handler(void *cls, struct MHD_Connection *connection,
 		goto parsingdone;
 	}
 
+        /* CARBYNE-SHC start */
+	if(session_path != NULL && !strcasecmp(session_path, "sanityhealthcheck"))
```

Comment:
SanityHealthCheck

Reply:
the path of the sanity health check today is 
/janus/sanityhealthcheck.
The '/janus' part is configured in the configuration file or inside the api.
So if I'll change it it will be mixed upper-lower cases => /janus/SanityHealthCheck.

Do you want that I'll try to make it be Upper case like /Janus/SanityHealthCheck or leave it as it is?
I don't know how it will inflict other API calls.